modules/core/src/main/java/org/apache/ignite/internal/management/StateCommand.java

```diff
@@ -1,4 +1,4 @@
-/*
+ /*
```

Comment:
Please remove this change

Reply:
The change is removed.

Reply:
Pattern/Matcher mechanism for checking output strings is implemented.